src/com/hms_networks/americas/sc/string/StringUtils.java

```diff
@@ -45,4 +48,35 @@ public static String replace(String string, String target, String replacement) {
     // Return modified string
     return stringBuffer.toString();
   }
+
+  /**
+   * Method to split the supplied string into smaller strings using the supplied regex as the
+   * delimiter.
+   *
+   * @param string string to split
+   * @param regex string split delimiter
+   * @return list of string split parts
+   * @since 1.2
+   */
+  public static List split(String string, String regex) {
```

Comment:
I forget but can you not return String[] on the flexy?

Reply:
You can. It would not be efficient to use a string array here though because we don't know the number of occurrences of `regex` and I didn't want to waste the CPU power converting a list to array